packages/compass-telemetry/src/telemetry-events.ts

```diff
@@ -2876,6 +2876,13 @@ type CreateIndexStrategiesDocumentationClicked = CommonEvent<{
   };
 }>;
 
+type DataModelingDiagramCreated = CommonEvent<{
```

Comment:
You probably want to add the `@category` to this event as well, so that it shows up in tracking plan something [like this](https://github.com/mongodb-js/compass/blob/main/docs/tracking-plan.md#explain-1) or else it would land on `Other` events